### PR TITLE
delete unnecessary step and update reference links

### LINKED ISSUE #1607: GlusterFS through CTDB - out of date links and recommendations
Please review recommended changes submitted in Rackspace Community Article: [https://community.rackspace.com/feedback/f/68/t/7974](https://community.rackspace.com/feedback/f/68/t/7974)

> Some of the information at https://support.rackspace.com/how-to/glusterfs-high-availability-through-ctdb/ is out-of-date:
> 
> Point (6) in the "Install CTDB" section is out-of-date.  Setting the "private dir" option has not been required for a long time.
> In "References" the link to http://ctdb.samba.org/testing.html is stale.  We're in the process of moving documentation to the wiki, which means that out-of-date documentation like this page has gone.  Better off linking to https://wiki.samba.org/index.php/CTDB_and_Clustered_Samba

### PR DESCRIPTION
**Pull request by stephamon**
Fixes #1607

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b5135bcb3b/how-to/)

File: content/cloud-servers/glusterfs-high-availability-through-ctdb.md [hunk 1]
```diff
@@ -124,7 +116,7 @@ The following items are necessary for CTDB installation:
           10.0.0.12/24 eth2
           10.0.0.13/24 eth2
 
-10.	In **/etc/ctdb/nodes**, list the server nodes that will be members of the CTDB cluster:
+9.	In **/etc/ctdb/nodes**, list the server nodes that will be members of the CTDB cluster:
 
            vi /etc/ctdb/nodes
```
Comment (nimbinatus) [inline content/cloud-servers/glusterfs-high-availability-through-ctdb.md]: So I can't comment where I want to, but on line 149/141, the order of the code list does not match the order of the unordered list below it.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8b0eeaceea/how-to/).

**Comment by nimbinatus** [comment]
LGTM, +1. Merging

**Pull request merged**